vis_sink.py

```diff
@@ -38,72 +38,63 @@
 
 from arcospyu.control.control_loop import Controlloop
 
-sys.path.append('../hand_cart')
-sys.path.append('../config_data/lwr')
+sys.path.append("../hand_cart")
+sys.path.append("../config_data/lwr")
 
 
 def configParse(argv):
     optparse.SUPPRESS_HELP
     parser = optparse.OptionParser(
-        'usage: %prog [options]', add_help_option=False)
-    parser.add_option('-e', '--help', action='help')
+        "usage: %prog [options]", add_help_option=False)
+    parser.add_option("-h", "--help", action="help")
     parser.add_option(
-        '-r',
-        '--robot',
-        dest='robot',
-        default='lwr',
-        type='string',
-        help='robot name')
+        "-r",
+        "--robot",
+        dest="robot",
+        default="lwr",
+        type="string",
+        help="".join([
+            "Tell the robot name so that the correct Articulated",
+            "Tree is loaded"
+        ]))
     parser.add_option(
-        '--arm_left',
-        action='store_true',
-        dest='arm_left',
+        "--arm_left",
+        action="store_true",
+        dest="arm_left",
         default=False,
-        help='arm left side')
+        help="Enable the configuration and simulation of a left arm")
     parser.add_option(
-        '--arm_right',
-        action='store_true',
-        dest='arm_right',
+        "--arm_right",
+        action="store_true",
+        dest="arm_right",
         default=False,
-        help='arm right side')
+        help="Enable the configuration and simulation or a right arm")
```

Comment:
Typo here should be:  simulation **of**a hand. Just commenting since it is easy to miss.